ruby/rspec/style_guide.md

```diff
@@ -0,0 +1,645 @@
+# RSpec style guide
+
+This RSpec style guide outlines our recommended best practices so that our developers can write code that can be maintained by other (future) developers.
+This is meant to be a style guide that reflects real-world usage, as well as a
+guide that holds to an ideal that has been agreed upon by many of the people it was intended to be used by.
+
+
+## Style Guide Rules
+
+### Line Returns after `feature`, `context`, or `describe`
+
+Do not leave line returns after `feature`, `context` or `describe`
+descriptions. It does makes the code more difficult to read and lowers the
```

Comment:
On: "It does makes the code more..." 
The "does" should be used to emphasize a point against another statement; but if used, then the following verb should be in root form (in this case "make").
E.g.: "It does make the code easier to read, but I would still avoid using it."